specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -10112,6 +10125,10 @@
         "modelAsString": true
       }
     },
+    "spotRestorePolicy": {
```

Comment:
This does not add this property to the VMSS object, or any other object for that matter. I suggest making the change in a VMSS object that is most relevant for you. This could be https://github.com/Azure/azure-rest-api-specs/blob/dfceec5c4d12b60821e88740c1fdd789685c9f26/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json#L11393

Reply:
This is just creating another type of name spotRestorePolicy, I think causing the Swagger LintDiff error that you need to resolve.

Reply:
Fixed